pkgs/development/tools/misc/dlang-dfmt/mkDub.nix

```diff
@@ -0,0 +1,141 @@
+{ pkgs ? import <nixpkgs> { }, stdenv ? pkgs.stdenv, lib ? pkgs.lib
```

Comment:
This file contains a builder, and builders are normally located in `pkgs/build-support`, so that they are available for everyone who needs to create packages based on this technology.

Introducing a builder is a quite significant change, as it impacts so many people which will use them to package software in the future. I believe this warrants its own PR. I think you should split it up, and try getting the builder PR merged first.